addons/hr/static/tests/m2x_avatar_employee.test.js

```diff
@@ -0,0 +1,541 @@
+        {
+            name: "Mario",
+            user_id: userId_1,
+            user_partner_id: partnerId_1,
+            work_email: "Mario@partner.com",
+        },
+        {
+            name: "Luigi",
+            user_id: userId_2,
+            user_partner_id: partnerId_2,
+        },
+    ]);
+    env["m2x.avatar.employee"].create([
+        {
+            employee_id: employeeId_1,
+            employee_ids: [employeeId_1, employeeId_2],
+        },
+        { employee_id: employeeId_2 },
+        { employee_id: employeeId_1 },
+    ]);
+    await start();
+    onRpc("has_group", () => false);
+    await mountView({
+        type: "list",
+        resModel: "m2x.avatar.employee",
+        arch: `<list><field name="employee_id" widget="many2one_avatar_employee"/></list>`,
+    });
+    expect(queryFirst(".o_data_cell div[name='employee_id']").innerText).toBe("Mario");
+    expect(queryAll(".o_data_cell div[name='employee_id']")[1].innerText).toBe("Luigi");
+    expect(queryAll(".o_data_cell div[name='employee_id']")[2].innerText).toBe("Mario");
+
+    // click on first employee avatar
+    await contains(queryAll(".o_data_cell .o_m2o_avatar > img")[0]).click();
+    await contains(".o_avatar_card");
+    expect(".o_card_user_infos > span").toHaveText("Mario");
+    expect(".o_card_user_infos > a").toHaveText("Mario@partner.com");
+    expect(queryFirst(".o_avatar_card_buttons button").textContent).toBe("Send message");
+    await contains(queryFirst(".o_avatar_card_buttons button")).click();
+    await contains(".o-mail-ChatWindow");
+    expect(
+        queryOne(
+            ".o-mail-ChatWindow-header button.o-dropdown.o-mail-ChatWindow-command > .text-truncate"
+        ).innerText
+    ).toBe("Mario");
+
+    // click on second employee
+    await contains(queryAll(".o_data_cell .o_m2o_avatar > img")[1]).click();
+    expect(".o_card_user_infos span").toHaveText("Luigi");
+    expect(".o_avatar_card").toHaveCount(1);
+    expect(queryFirst(".o_avatar_card_buttons button").textContent).toBe("Send message");
+    await contains(queryFirst(".o_avatar_card_buttons button")).click();
+    expect(".o-mail-ChatWindow").toHaveCount(2);
+    expect(
+        queryAll(
+            ".o-mail-ChatWindow-header button.o-dropdown.o-mail-ChatWindow-command > .text-truncate"
+        )[0].textContent
+    ).toBe("Luigi");
+
+    // click on third employee (same as first)
+    await contains(queryAll(".o_data_cell .o_m2o_avatar > img")[2]).click();
+    expect(".o_card_user_infos span").toHaveText("Mario");
+    expect(".o_avatar_card").toHaveCount(1);
+    expect(queryOne(".o_card_user_infos span").textContent).toBe("Mario");
+    expect(".o_card_user_infos > a").toHaveText("Mario@partner.com");
+    expect(queryFirst(".o_avatar_card_buttons button").textContent).toBe("Send message");
+    await contains(queryFirst(".o_avatar_card_buttons button")).click();
+    expect(".o-mail-ChatWindow").toHaveCount(2);
```

Comment:
`contains(queryFirst(selector)).click()` => `contains(selector:eq(0)).click()`

Adapt equivalent occurrences in the whole file.